### ISSUE TITLE
experiment with line wrapping for long alleles

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1436]
Here's how it looks if we allow wrapping, including after commas:

![phenotype_annotation_table_2](https://user-images.githubusercontent.com/90474/39342377-e52b51fc-4a2b-11e8-975f-667c2028e110.png)

We can allow wrapping just before the "[" in the expression.  It doesn't look too bad:

![phenotype_annotation_table_3](https://user-images.githubusercontent.com/90474/39343433-dd07e0d0-4a30-11e8-9100-35ab9576ab08.png)

**Comment by ValWood** [comment]
@Antonialock 
not sure, maybe don't allow wrapping fr  multi-allele genotypes?

@mah11 ditto - what do multi-allele genotypes look like with wrapping?

**Comment by kimrutherford** [comment]
> what do multi-allele genotypes look like with wrapping?

They look ... not great.  It's hard to see which allele is which and the gene names don't align nicely.

![multi-allele-wrapped-1](https://user-images.githubusercontent.com/90474/40095953-af52bace-5922-11e8-8ebe-22a499bd6154.png) 

We can separate the alleles to help the first problem:

![multi-allele-wrapped-2](https://user-images.githubusercontent.com/90474/40095959-b486aa96-5922-11e8-81d3-f302a472410d.png)

The gene alignment can be fixed too if we go this route.

**Comment by ValWood** [comment]
Yeah not good. I think if we need to separate  the alleles it defeats tho object  a bit.

Let's keep thinking on this one...

**Comment by ValWood** [comment]
implement truncating the long alleles with "........"  at  60 chars 
have the full allele description available by mouse over
test with curs/7803457eba12e494 (multi alleles as a test)

**Comment by kimrutherford** [comment]
I've made a change to truncate the long allele names.

I'll release the change once #1539 is fixed.

**Comment by kimrutherford** [comment]
> I'll release the change once #1539 is fixed.

I re-released this morning which is what caused #1540

**Comment by ValWood** [comment]
I still see

![did you](https://user-images.githubusercontent.com/7359272/42227817-41ea2520-7eda-11e8-91eb-5263f5e2b0fe.jpg)

**Comment by kimrutherford** [comment]
> I still see

Sorry about that.  I had missed the cases when the session was read-only and when the allele names are from Chado.  Should be OK now.

**Comment by ValWood** [comment]
I think that works really well. In my worst case  scenario:

![wcs](https://user-images.githubusercontent.com/7359272/42266164-7ac16596-7f6d-11e8-9b28-0fdcf057e75f.jpg)


We'll stay with this  for a while, but if we need more space we  can reduce the string length still further

**Comment by ValWood** [comment]
One possible problem might be not being able to see the expression?

Maybe the .... ing should be in the middle

i.e     
pds5-him(aaP82A,D83A ......   [WT product level]
top2-12KR(aaK1228,K1254,K1255 ...  [WT product level]

@Antonialock 
@mah11

**Comment by ValWood** [comment]
Question @mah11 @Antonialock  should we move the dots?

**Comment by ValWood** [comment]
This  is a closed ticket. Question transferred to
https://github.com/pombase/canto/issues/1587
